docs/source/user-guide/vision/vision_properties.rst

```diff
@@ -0,0 +1,188 @@
+===============
+Data Properties
+===============
+
+Some of the Deepchecks' checks (e.g. train-test drift) use "Properties". Properties are simple one-dimension values that
+are extracted either from the images, labels or predictions of the data.
+
+For example, one image property is **brightness**, and one label property is **bounding box area** (for detection
+tasks). Each property produces a single value per image or per individual label in the image.
+
+What Are Properties Used For?
+=============================
+
+The properties values' distribution is a characteristic of your data that might have hidden a relation to your
+model performance. They might come handy in a few situations:
+
+1. New unlabeled data: To approximate how your model will work on the new data, since you don't have labels
+   in order to do a performance check. If the new data's properties are different by a large margin from the
+   train's properties, then it might indicate the model will have harder time inferring on it.
+2. Low test performance: If your model is having low performance on your test data, a good step in debugging
+   it is to check the drift of different properties between the train and the test. If there are some
+   properties with a high drift it might help you pinpoint to a relevant causes of the model's low performance.
```

Comment:
I don't see the difference between 1 and 2.

Reply:
1 is for unlabeled data, 2 is for labeled data